### PR TITLE
Fix for unresolved MigrationConstants

### LINKED ISSUE #1363: Bug: references to MigrationConstants
The constant MigrationConstants was renamed to CommonConstants as part of this work:

https://github.com/ualbertalib/HydraNorth/pull/1360

Unfortunately some of the migration rake tasks still refer to MigrationConstants. I will submit a new PR with a fix.

### PR DESCRIPTION
**Pull request by cwant**
MigrationConstants was renamed to CommonConstants, but several references to the old name still remained.

Fixes #1363.

### CODE REVIEW

**Comment by pbinkley** [comment]
:+1:

File: lib/tasks/dataverse_migration.rake [hunk 1]
```diff
@@ -55,26 +55,26 @@ namespace :migration do
       MigrationLogger.info "Processing the object #{object_id}"
       #reading the metadata file
       metadata_file = Nokogiri::XML(File.open(file))
-      metadata = metadata_file.xpath("//ddi_to_dcterms", MigrationConstants::NS)
+      metadata = metadata_file.xpath("//ddi_to_dcterms", CommonConstants::NS)
```
Comment (mbarnett) [inline lib/tasks/dataverse_migration.rake]: Where does CommonConstants come from, just so I understand the context of this change?

Comment (cwant) [inline lib/tasks/dataverse_migration.rake]: CommonConstants is here:

https://github.com/ualbertalib/HydraNorth/blob/master/lib/tasks/constants/common_constants.rb

It looks like MigrationConstants and some other constants were merged into it as part of this commit:

https://github.com/ualbertalib/HydraNorth/commit/119acc300fae5514824bffd6d4b6bd301a607d43#diff-178d2afef07858d1f2e00cb3662df5bcR1

Comment (mbarnett) [inline lib/tasks/dataverse_migration.rake]: Aha, thanks! LGTM :shipit: etc

**Pull request merged**